src/elf_parser/mod.rs

```diff
@@ -9,7 +9,7 @@ use crate::{ArithmeticOverflow, ErrCheckedArithmetic};
 use {consts::*, types::*};
 
 const SECTION_NAME_LENGTH_MAXIMUM: usize = 16;
-const SYMBOL_NAME_LENGTH_MAXIMUM: usize = 1024;
+const SYMBOL_NAME_LENGTH_MAXIMUM: usize = 64;
```

Comment:
This change makes it impossible to load files generated by our Move compiler. For example, this relocation triggers Unknown symbol error.
```
0000000000000578  000000250000000a R_BPF_64_32            000000000008d940 _ZN4core3fmt3num53_$LT$impl$u20$core..fmt..UpperHex$u20$for$u20$u16$GT$3fmt17hda2568c893a78314E
```
The symbol is 95 characters long. There are longer symbols in Move compiled modules too.